src/Composer/Package/Version/VersionGuesser.php

```diff
@@ -63,26 +63,28 @@ public function __construct(Config $config, ProcessExecutor $process, SemverVers
      */
     public function guessVersion(array $packageConfig, $path)
     {
-        if (function_exists('proc_open')) {
-            $versionData = $this->guessGitVersion($packageConfig, $path);
-            if (null !== $versionData && null !== $versionData['version']) {
-                return $this->postprocess($versionData);
-            }
+        if (!function_exists('proc_open')) {
+            return;
```

Comment:
Should we explicitly return `null` here?

```suggestion
            return null;
```